scripts/arangodb-bash-setup/arangodb-bash-setup.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -e
+
+# IMPORTANT: Consult sample.env in apps/api/src/app/config for 
+# environment configuration prior to running this script
+
+# Check if this bash setup script is being run within it's own directory
+VALID_EXECUTE_DIRECTORY="coscrad/scripts/arangodb-bash-setup"
```

Comment:
I'm not entirely sure this is what we want. I feel like we should be able to grab just the script and `test-data.json` from the repo and nothing else and still be ready to roll.